src/doc-utils.tsx

```diff
@@ -467,12 +467,21 @@ export function DocsMarkdown({ fileSrc }: { fileSrc: string }) {
                 </Tag>
               );
```

Comment:
I do not know what we use the extra id for in the heading but I think that we also need to change the 'heading' to the 'h1-h5' tags as they have changed aswell.
See
https://github.com/remarkjs/react-markdown/blob/main/changelog.md#change-renderers-to-components
https://github.com/remarkjs/react-markdown#appendix-b-components

Reply:
I just tried to comment out the id. Seemingly, it doesn't change anything visually. And if we no longer need the id, we no longer need to specify any markdown for h1-h6, thus we can remove it all together.